### DISCUSSION TITLE
environment variables to fill the public dashboard or the admin dashboard

### DISCUSSION THREAD

**Discussion by henkiewie** [alexjustesen/speedtest-tracker#1585] [Ideas]
I would like to see an environment variable to fill the dashboard with the choices in the picture. Now the last 24 hour is the default.
<img width="195" alt="Screenshot 2024-06-28 at 08 28 38" src="https://github.com/alexjustesen/speedtest-tracker/assets/74558874/37894974-5f58-4cba-b21b-87ca1952bdee">

Greetings Henk.

**Comment by svenvg93** [comment]
There some changes planned for the time ranges. See https://github.com/alexjustesen/speedtest-tracker/issues/738 and https://github.com/alexjustesen/speedtest-tracker/issues/448 for more information and updates.

**Reply by henkiewie** [reply]
Ok sounds good, but just adding an extra environment variable maybe does not have to wait for an whole rewrite of the program?
When the rewrite is happening however, maybe a graph like grafana has would be nice so one can even zoom in.
